test/unit/warp/src/Activemask.cpp

```diff
@@ -51,7 +51,9 @@ class ActivemaskMultipleThreadWarpTestKernel
 
         auto const actual = alpaka::warp::activemask(acc);
         using Result = decltype(actual);
-        Result const allActive = (Result{1} << static_cast<Result>(warpExtent)) - 1;
+        Result const allActive = static_cast<size_t>(warpExtent) == sizeof(Result) * 8u
+            ? ~static_cast<Result>(0u)
```

Comment:
```suggestion
            ? ~Result{0}
```
I think this should be the same?